docs/SUBMISSION.md

```diff
@@ -21,7 +21,13 @@ Please refer to the inline comments in the base `application.yaml` configuration
 
 ## TAN Verification
 
-Coming soon
+When submitting diagnosis keys, a Transaction Authorization Number (TAN) token must be present in the request header section (`cwa-authorization`).
+Before delegating the TAN validation to the verification server, the TAN is verified to be an UUID on the Submission Service side.
```

Comment:
maybe use capital letters for system components . I.e. Verification Server . Probably worth doing only if the rest of the document also uses capital letters.

Reply:
"to be a UUID"

Reply:
Done. 😄